paasta_tools/cli/cmds/status.py

```diff
@@ -259,7 +261,24 @@ def report_status_for_cluster(
                 for line in status.rstrip().split('\n'):
                     paasta_print('    %s' % line)
 
-    paasta_print(report_invalid_whitelist_values(instance_whitelist, seen_instances, 'instance'))
+            paasta_print('Remote runs:')
```

Comment:
they do currently see list of chronos jobs. remote runs will for now be reflecting whatever is running via tron. but i do generally agree that usefulness of this is questionable.

as for too much ssh'ing - what's your opinion of paramiko? it could help with ssh overhead by having a channel to execute commands instead of shelling out.